controls/views.py

```diff
@@ -178,6 +178,81 @@ def controls_selected(request, system_id):
         # User does not have permission to this system
         raise Http404
 
+@login_required
+def system_controls_add(request, system_id):
+    """Add a selected control to a system (e.g., selected controls)"""
+
+    # Get control values from request.POST
+    catalog_key = request.POST['catalog_key'].replace(" ","_") # Make sure catalog key has underscores instead of spaces
+    control_id = request.POST['control_id']
```

Comment:
```suggestion
    control_id, catalog_key = request.POST['control_id_catalogkey'].split('-_-')
    catalog_key = catalog_key.replace(" ", "_")  # Make sure catalog key has underscores instead of spaces

```

Reply:
See previous discussion of the `controls_selected.html` recommended changes.